parsons/mobilize_america/ma.py

```diff
@@ -162,16 +178,20 @@ def get_events(self, organization_id=None, updated_since=None, timeslot_start=No
                 timeslots_tbl = tbl.long_table(['id'], 'timeslots', 'event_id')
                 return {'events': tbl, 'timeslots': timeslots_tbl}
 
+            elif max_timeslots == 0:
+                tbl.remove_column('timeslots')
+
             else:
                 tbl.unpack_list('timeslots', replace=True, max_columns=max_timeslots)
                 cols = tbl.columns
                 for c in cols:
                     if re.search('timeslots', c, re.IGNORECASE) is not None:
                         tbl.unpack_dict(c)
+                        tbl.materialize()
 
         return tbl
 
-    def get_events_organization(self, organization_id=None, updated_since=None, timeslot_start=None,
+    def get_events_organization(self, organization_id, updated_since=None, timeslot_start=None,
```

Comment:
Before what this function did was the exact same as `get_events` (the connector was a little confused), and is deprecated now. Before if you didn't pass an `organization_id`, it would return all public events in all of Mobilize made by anyone.

The endpoint that you use now to get all of the events for an organization does need an organization ID, so if `organization_id` were `None` then it would always error with the current API. So in a sense we are making a breaking change, but Mobilize made a breaking change to their API so there's nothing we can do.

Another reason to change it this way is that the method is clearly meant to get all of the events for a particular organization. If someone is using this method instead of `get_events`, that's clearly what they want to do. So if they're using `get_events_organization` without an `organization_id` their script is already failing, since it's not working the way they think it is. In my opinion it's better to just fail their script and make them realize their mistake.

Reply:
Gotcha, this makes sense. I added the breaking changes label, we'll just flag it in the release notes.